set.mm

```diff
@@ -502199,6 +502199,64 @@ The sumset operation can be used for both group (additive) operations and
       JXLUUPUUOVEKYOGFXKVGXOXQXRXSUAAYPCYMEYFUBUCJHUWTUXAWNXTYA $.
   $}
 
+  ${
+    $d .0. x y z $.  $d B x y z $.  $d R x y z $.  $d U x y z $.
+    drngidl.b $e |- B = ( Base ` R ) $.
+    drngidl.z $e |- .0. = ( 0g ` R ) $.
+    drngidl.u $e |- U = ( LIdeal ` R ) $.
+    $( A ring is a division ring if and only if its only ideals are the zero
+       ideal and the improper ideal.  (Proposed by Gerard Lang, 13-Mar-2025.)
```

Comment:
I think we call the whole (base set of the) ring a "unit ideal" (see, e.g., ~lidl1el and ~zarcls1), and not an "improper ideal".

Reply:
Yes, even though there are *[some](https://ncatlab.org/nlab/show/improper+ideal)* references to that term, it does not seem to be widely used and it's better to have a uniform naming throughout set.mm.
I'll change it.

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4712/commits/d99eb52b932981fe7662baf64937af731755c57b